apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -5771,8 +5771,8 @@
   "Toolbox_room_actions": "Primary Room actions",
   "Theme_light": "Light",
   "Theme_dark": "Dark",
-  "Enable_of_limit_apps_currently_enabled": "**{{enabled}} of {{limit}} {{context}} apps currently enabled.**\n\nWorkspaces on Community Edition can have up to {{limit}} {{context}} apps enabled.\n\n**{{appName}} will be disabled by default.** Disable another {{context}} app or upgrade to Enterprise to enable this app.",
-  "Enable_of_limit_apps_currently_enabled_exceeded": "**{{enabled}} of {{limit}} {{context}} apps currently enabled.**\n\nCommunity edition app limit has been exceeded.\n\nWorkspaces on Community Edition can have up to {{limit}} {{context}} apps enabled.\n\n**{{appName}} will be disabled by default.** You will need to disable at least {{exceed}} other {{context}} apps or upgrade to Enterprise to enable this app.",
+  "Enable_of_limit_apps_currently_enabled": "**{{enabled}} of {{limit}} {{context}} apps currently enabled.**  \n  \nWorkspaces on Community Edition can have up to {{limit}} {{context}} apps enabled.  \n  \n**{{appname}} will be disabled by default.** Disable another {{context}} app or upgrade to Enterprise to enable this app.",
+  "Enable_of_limit_apps_currently_enabled_exceeded": "**{{enabled}} of {{limit}} {{context}} apps currently enabled.**  \n  \nCommunity edition app limit has been exceeded.  \n  \nWorkspaces on Community Edition can have up to {{limit}} {{context}} apps enabled.  \n  \n**{{appname}} will be disabled by default.** You will need to disable at least {{exceed}} other {{context}} apps or upgrade to Enterprise to enable this app.",
```

Comment:
actually, won't this case shift in appname (from appName) break those translations? there are other places I see this changed.

Reply:
You are right, just fixed